test/test_autograd.py

```diff
@@ -1067,6 +1067,8 @@ def prod_single_zero(dim_size):
     (Triu, (), ((S, S),)),
     (Triu, (2,), ((S, S),), 'idx'),
     (Trace, (), ((S, S),)),
+    (Cross, (-1,), ((S, 3), (S, 3))),
```

Comment:
-1 is the default behaviour for `torch.cross` and `variable.cross` , and I will also make it default of `Cross`